static/versionDetails.json

```diff
@@ -404,6 +404,11 @@
       "flag": "licenseURL",
       "description": "A URL to a licensing agreement text file that will be presented during installation and needs to be accepted for the install to continue.",
       "type": "string"
+    },
+    {
+      "flag": "hostPreflights",
+      "description": "A troubleshoot.sh/v1beta2 spec for adding additional preflight checks prior to a kURL installation. Information on writing preflight checks can be found at https://troubleshoot.sh/docs/preflight/introduction/.",
```

Comment:
Does the Troubleshoot site give information on the `HostPreflight` kind? I'm not sure that it does. I am not confident that a vendor could come to this page and figure this out on their own. There are lots of collectors supported for host preflights, but I don't know that they're documented anywhere. If they are, we need to link to that so that people can see what their options are.

Reply:
I couldn't find any documentation on that. We could link to the kURL github examples of HostPreflights.